app/models/nic/base.rb

```diff
@@ -144,16 +150,30 @@ def host_managed?
       self.host && self.host.managed? && SETTINGS[:unattended]
     end
 
-    def require_ip_validation?
+    def require_ip_validation?(ip_field, other_ip_field, subnet)
+      raise Foreman::Exception.new(N_("invalid ip field %s") % ip_field) unless ip_field == :ip || ip_field == :ip6
+      raise Foreman::Exception.new(N_("invalid other ip field %s") % other_ip_field) unless other_ip_field == :ip || other_ip_field == :ip6
```

Comment:
@lzap : Sorry, I don't see it... :-( What's your point exactly? Do you want to translate "ip_field"? Do we need to translate this at all? A user should never see this.